### DISCUSSION TITLE
Url For Webhooks saying invalid

### DISCUSSION THREAD

**Discussion by GreedyGitHub** [RevertIT/mybb-rt_discord_webhooks#7] [Q&A] [answered]
Is there a special format to do the webhook, if i copy the url from the discord webhook its saying its invalid for some unkown reason am i doing something wrong?

**Comment by GreedyGitHub** [comment]
Invalid webhook format specified

**Comment by RevertIT** [comment]
Only if discord changed the url structure. Its checked via regular
expression.

**Reply by GreedyGitHub** [reply]
This is the webhook im trying to add

https://discordapp.com/api/webhooks/1163040954383597598/COIWOdIEsJFuKNvsPwhvWtc0HIMVfFKt46OsGyTYd1hFaEsTjO6fxZvh9urRk_TuV9ja

Saying invalid url format.

**Comment by RevertIT** [comment, marked as answer]
Use discord.com instead of discordapp.com
